### ISSUE TITLE
MapId change is not working 

### ISSUE THREAD

**Issue by padawannn** [visgl/react-google-maps#4]
The first time that you try to change the Map ID you get this warning
```
Google Maps JavaScript API: A Map's mapId property cannot be changed after initial Map render 
```
but the change happens. The second time that you try it, it doesn't work.

I think that the problem comes from this [line](https://github.com/visgl/react-google-maps/blob/main/src/components/map.tsx#L146) that prevents a new instance of the map from being created

**Comment by usefulthink** [comment]
This should be fixed in d97cae9ded0de30604e543d78341984cb61de942. I also added [a new example for this](https://visgl.github.io/react-google-maps/examples/change-map-id).

**Comment by usefulthink** [comment]
A fix is now available in version 0.0.4 on npm.
